reference_controllers/rabbitmq_controller_unverified/src/statefulset.rs

```diff
@@ -0,0 +1,480 @@
+                                name: Some(rabbitmq.metadata.name.clone().unwrap() + "-default-user"),
+                                items: Some(vec![
+                                    corev1::KeyToPath{
+                                        key: "default_user.conf".to_string(),
+                                        path: "default_user.conf".to_string(),
+                                        ..corev1::KeyToPath::default()
+                                    },
+                                ]),
+                                ..corev1::SecretProjection::default()
+                            }),
+                            ..corev1::VolumeProjection::default()
+                        }
+                    ]),
+                    ..corev1::ProjectedVolumeSource::default()
+                }),
+                ..corev1::Volume::default()
+            },
+            corev1::Volume{
+                name:"rabbitmq-erlang-cookie".to_string(),
+                empty_dir: Some(corev1::EmptyDirVolumeSource{ ..corev1::EmptyDirVolumeSource::default() }),
+                ..corev1::Volume::default()
+            },
+            corev1::Volume{
+                name:"erlang-cookie-secret".to_string(),
+                secret: Some(corev1::SecretVolumeSource{
+                    secret_name: Some(rabbitmq.metadata.name.clone().unwrap() + "-erlang-cookie"),
+                    ..corev1::SecretVolumeSource::default()
+                }),
+                ..corev1::Volume::default()
+            },
+            corev1::Volume{
+                name:"rabbitmq-plugins".to_string(),
+                empty_dir: Some(corev1::EmptyDirVolumeSource{ ..corev1::EmptyDirVolumeSource::default() }),
+                ..corev1::Volume::default()
+            },
+            corev1::Volume{
+                name: "pod-info".to_string(),
+                downward_api: Some(corev1::DownwardAPIVolumeSource{
+                    items: Some(vec![
+                        corev1::DownwardAPIVolumeFile{
+                            path: "skipPreStopChecks".to_string(),
+                            field_ref: Some(corev1::ObjectFieldSelector{
+                                field_path: "metadata.labels['skipPreStopChecks']".to_string(),
+                                ..corev1::ObjectFieldSelector::default()
+                            }),
+                            ..corev1::DownwardAPIVolumeFile::default()
+                            },   
+                        ]),
+                    ..corev1::DownwardAPIVolumeSource::default()
+                }),
+                ..corev1::Volume::default()
+            }
+    ];
+
+    let rbmq_container_volume_mounts = vec![
+        corev1::VolumeMount{
+            name: "rabbitmq-erlang-cookie".to_string(),
+            mount_path: "/var/lib/rabbitmq/".to_string(),
+            ..corev1::VolumeMount::default()
+        },
+        corev1::VolumeMount{
+            name: "persistence".to_string(),
```

Comment:
I have a dump question: what is the "persistence" volume? I don't see it in `let volumes = vec![...]`.